src/movies.js

```diff
@@ -1,22 +1,70 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+    const allDirectors = [];
+    moviesArray.map((movie) => allDirectors.push(movie.director));
+    return allDirectors
+}
```

Comment:
Esto esta bien pensado eeee!!
Pero es mas sencillo, te lo voy a modificar un poco
```suggestion
function getAllDirectors(moviesArray) {
    return moviesArray.map((movie) => movie.director);
}
```

Bien que hacemos aqui, si te fijas lo que devuelve el metodo map es en si un array, por ello podemos devolver el resultado del metodo, y asi no ahorramos generar una nueva constante! :D